### DISCUSSION TITLE
Data types

### DISCUSSION THREAD

**Discussion by Sreekar2610** [Sreekar2610/python#11] [Q&A] [answered]
### What is the difference between a Mutable datatype and an Immutable data type?

**Comment by SandhyaMadhunagula** [comment]
The mutable objects can be changed to any value or state without adding a new object. Whereas, the immutable objects can not be changed to its value or state once it is created.

**Comment by rahulhugar5** [comment, marked as answer]
The mutable objects can be changed to any value or state without adding a new object. Whereas, the immutable objects can not be changed to its value or state once it is created.
